nlpsandboxclient/validation.py

```diff
@@ -0,0 +1,44 @@
+"""Validation of data and platform components like NLP Tools"""
+
+from jsonschema import Draft7Validator
+
+from .client import NlpClient
+
+
+class NlpToolClient(NlpClient):
+    """Client for developed Nlp Tools"""
+    def __init__(self, tool_type, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.tool_type = tool_type
+
+    def get_tool(self):
+        """Get tool schema"""
+        return self.rest_get("/tool")
+
+    def validate(self):
+        """Validate schema"""
+        # Check schema is correct first
+        if self.get_ui().status_code != 200:
+            raise ValueError("/ui endpoint not configured")
+        if self.get_health().get('status') != 'pass':
+            raise ValueError("/health endpoint not configured")
+        schema = self.get_tool()
+        Draft7Validator.check_schema(schema)
```

Comment:
Is this schema the user specified schema?

Reply:
I think that you have figured this out in your next commit. The JSON Schema used to validate data will be available in https://github.com/Sage-Bionetworks/nlp-sandbox-schemas.